# 我队增益类型
uint8 recoverybuff
uint8 coolingbuff
uint8 defencebuff
uint8 vulnerabilitybuff
uint16 attackbuff
uint8 remainingenergy